zio-http/src/main/scala/zhttp/http/Scheme.scala

```diff
@@ -48,6 +48,13 @@ object Scheme       {
     case _       => None
   }
 
+  def decode2(scheme: String): Scheme = scheme.length match {
```

Comment:
this implementation can introduce bugs

Reply:
- Should be called as unsafeDecode
- Should be package private